rpcs3/util/sysinfo.cpp

```diff
@@ -414,15 +414,24 @@ std::string utils::get_firmware_version()
 
 		version = version.substr(start, end - start);
 
-		// Trim version
+		// Trim version (e.g. '04.8900' becomes '4.89')
 		const usz trim_start = version.find_first_not_of('0');
-		const usz trim_end = version.find_last_not_of('0');
```

Comment:
With trim_start, limit the range to 1 character instead of the whole string character so 00.6666 can also work.